data/json/monsters.json

```diff
@@ -5729,6 +5765,7 @@
     "armor_bash":0,
     "armor_cut":0,
     "vision_day":30,
+    "vision_night":10,
```

Comment:
Ah, nothing like a joke char that's downright lethal to put people off.  Random night-murder seems unfun.